packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -5,11 +5,11 @@ outline: 'deep' # shows all h3 headings in outline in Vitepress
 
 # Flowcharts - Basic Syntax
 
-All Flowcharts are composed of **nodes**, the geometric shapes and **edges**, the arrows or lines. The mermaid code defines the way that these **nodes** and **edges** are made and interact.
+Flowcharts are composed of **nodes** (geometric shapes) and **edges** (arrows or lines). The Mermaid code defines how nodes and edges are made and accommodates different arrow types, multi-directional arrows, and any linking to and from subgraphs.
 
-It can also accommodate different arrow types, multi directional arrows, and linking to and from subgraphs.
-
-> **Important note**: Do not type the word "end" as a Flowchart node. Capitalize all or any one the letters to keep the flowchart from breaking, i.e, "End" or "END". Or you can apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897).
+```warning
+If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
+\``` Remove the \ here. Suggestion didn't include the ending triplequote.
```

Comment:
@huynhicode 😬 github UI wasn't showing the backticks. Can you fix this?

Reply:
Gah, I can see why using GitHub UI is not as ideal as making updates in an IDE.

Looks nice now - thanks for your help @sidharthv96 !

<img width="1496" alt="Screenshot 2023-05-02 at 11 42 33 PM" src="https://user-images.githubusercontent.com/35910788/235847578-011b5fb8-8e03-4d0b-a3b3-c04b853442cc.png">